README.md

```diff
@@ -178,17 +178,13 @@ For example, if your Iroha instance is running at `127.0.0.1:8080` and you want
 curl -X POST \
     -H 'content-type: application/json' \
     http://127.0.0.1:8080/configuration \
-    -d '{"LogLevel": "DEBUG"}' -i
+    -d '{"logger": {"level": "DEBUG"}}' -i
 ```
 </details>
 
-#### JSON Logging Mode
+Logger supports multiple output formats: `full` (default), `compact`, `pretty`, and `json`.
```

Comment:
I think this should mention the name of the setting changing the output format

Reply:
Done.